tower/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# 0.4.2 (January 11, 2021)
+
+- Fix internal `Semaphore` to be `Sync`, this affects all types that internally store
```

Comment:
maybe note specifically that this is Buffer and ConcurrencyLimit, instead of just talking about the implementation detail?
```suggestion
- Fix missing `Sync` implementation for `Buffer` and `ConcurrencyLimit`
```